software/glasgow/applet/program/mec16xx/__init__.py

```diff
@@ -190,6 +190,10 @@ def add_interact_arguments(cls, parser):
             "file", metavar="FILE", type=argparse.FileType("wb"),
             help="write EC firmware to FILE")
 
+        p_read = p_operation.add_parser(
+            "erase-flash", help="erase the flash memory only using normal flash controller " +
```

Comment:
I would go for:

```suggestion
            "erase", help="erase EC firmware (non-emergency mode)"
```

then reword the other one to

```suggestion
            "emergency-erase", help="erase EC firmware (emergency mode)")
```

and add a short description of the difference in the main help text, since it's a bit too long to be put inline in the operations.

Reply:
done